pandas/tools/plotting.py

```diff
@@ -359,7 +359,10 @@ def scatter_matrix(frame, alpha=0.5, figsize=None, ax=None, grid=False,
     density_kwds = density_kwds or {}
 
     # workaround because `c='b'` is hardcoded in matplotlibs scatter method
-    kwds.setdefault('c', plt.rcParams['patch.facecolor'])
+    if 'color' in kwds:
+        pass
+    else:
```

Comment:
``kwds.setdefault('c', kwdgs.pop('color', plt.rcParams['patch.facecolor']))`` is a big more idiomatic